apb-wrapper

```diff
@@ -7,9 +7,11 @@ fi
 if [ $UID == 0 ]; then
   ln -sf /.kube /root/.kube
 else
-  echo "apb:x:$UID:$UID:apb:/home/apb:/bin/bash" >> /etc/passwd
-  sudo groupadd apb -g $UID
-  sudo usermod -a -G apb apb
+  # ensure $HOME exists and is accessible by group 0 (we don't know what the runtime UID will be)
+  echo "apb:x:$UID:0:apb user:/home/apb:/sbin/nologin" >> /etc/passwd
```

Comment:
where did you get the `group 0` part? that's new to me :) 

I expected to see this change: https://github.com/ansibleplaybookbundle/apb-base/blob/master/files/usr/bin/entrypoint.sh#L43-L58

You will have to adapt the above patch to work with the apb-wrapper.